chocolatey/choco
# (maint) Remove chocolatey.benchmark and chocolatey.powershell from MSI build

## Description Of Changes

Remove building extra packages from the MSI configuration.

## Motivation and Context

Newly added projects were causing the MSI build to fail. This removes all projects for all MSI configurations except the `chocolatey.install` project.

## Testing

1. Tag a commit locally with `git commit 2.4.0`
2. Run `./build.official.bat`
3. Verify that there is an MSI created in `code_drop/Artificacts`

### Operating Systems Testing

Windows 10

## Change Types Made


## Change Checklist


## Related Issue

N/A
